### DISCUSSION TITLE
2024/3/5 答疑记录

### DISCUSSION THREAD

**Discussion by Okabe-Rintarou-0** [Okabe-Rintarou-0/BookStore-Frontend#16] [Q&A]
请今天参加答疑的同学有空的话帮忙记录一下自己的问题，和对应的解决方案。

**Comment by Okabe-Rintarou-0** [comment]
css 引入问题：https://juejin.cn/post/7031528538209681444

**Reply by userKangkang** [reply]
当时问这个的原因其实是因为调tailwind css的时候出现了一个十分匪夷所思的bug，后面用内置`style={{...}}`才解决；然后就是tailwind css和ant design等组件库有个冲突的地方（比如ant design的button被设成透明的，以及一些诸如h1的标签失去了自己默认的属性），可以以下面方式解决
![1709654105688](https://github.com/Okabe-Rintarou-0/BookStore-Frontend/assets/114564389/2ad81e36-7fbf-4c51-84d9-004d0de52ed6)
看来这个tailwind css好像也没那么好用

**Comment by Okabe-Rintarou-0** [comment]
demo 是为了让大家明白前端是干什么的，跟着readme做一遍也就知道前端大概怎么开发，部署了
